File: Controls/EditorA/filepicker/src/main/java/com/kelfan/filepicker/FilterHidden.java
package com.kelfan.filepicker;

import java.io.File;
import java.io.FileFilter;
import java.io.Serializable;

/**
 * Created by Dimorinny on 31.05.16.
 */
public class FilterHidden implements FileFilter, Serializable {

    @Override
    public boolean accept(File f) {
        return !f.isHidden();
    }
}

File: MyProject/07KotlinEditor/app/src/main/java/com/kelfan/kotlineditor/filter/FilterHidden.kt
package com.kelfan.kotlineditor.filter

import java.io.File
import java.io.FileFilter
import java.io.Serializable

/**
 * Created by Administrator on 2/02/2018.
 */
class FilterHidden : FileFilter, Serializable {

    override fun accept(f: File): Boolean {
        return !f.isHidden
    }
}